book/src/contributing/architecture.md

```diff
@@ -0,0 +1,100 @@
+# Architecture
```

Comment:
May I know the relationship between this file and https://github.com/fzyzcjy/flutter_rust_bridge/blob/master/book/src/contributing/design.md?

Reply:
Good question, there is a fair amount of overlap between them but I don't have a good idea how to organize their contents together yet.

Reply:
A naive approach may be merging the two files